README.md

```diff
@@ -1 +1,30 @@
-# hw01
\ No newline at end of file
+# hw01
+
+### Brief Biography of William Graif
+
+I am finishing my at the University of Chicago. I am studying data science and public policy. I have been living in the same house near New York City since I was *nine months* old. I have also been both playing and teaching chess for a very long time. Here is me at K-1 Nationals:
+
+![](C:\Users\wgrai\OneDrive\Pictures\young_me.jpg)
```

Comment:
You need to push and commit the `young_me.jpg` onto your github as well. Then, you can't use your local path since github won't have the access to your `C:\Users\wgrai\OneDrive\Pictures`. You can simply do `![](young_me.jpg)`

Reply:
That makes sense. I'll know that for next time. Thank you!